### PR TITLE
Fix verify-crdgen.sh do not verify operator

### PR DESCRIPTION
**Pull request by Poor12**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Verify-crdgen.sh now do not verify the `karmada` crd of karmada-operator.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
None

### CODE REVIEW

File: hack/verify-crdgen.sh [hunk 1]
```diff
@@ -53,3 +58,13 @@ else
...
+
+diff -Naupr "${DIFFOPERATOR}" "${TMP_DIFFOPERATOR}" || ret=$?
+cp -a "${TMP_DIFFOPERATOR}"/* "${DIFFOPERATOR}"
+if [[ $ret -eq 0 ]]
+then
+  echo "${DIFFOPERATOR} up to date."
+else
+  echo "${DIFFOPERATOR} is out of date. Please run hack/update-crdgen.sh"
+  exit 1
+fi
```
Comment (XiShanYongYe-Chang) [inline hack/verify-crdgen.sh:70]: need a new blank line.

**Review by XiShanYongYe-Chang — COMMENTED** [review]
Do we need `make update`?

**Comment by calvin0327** [comment]
Sorry, I find the crd yaml of chart is out of date.  Thinks for this pr to verify this

**Comment by XiShanYongYe-Chang** [comment]
> Sorry, I find the crd yaml of chart is out of date. Thinks for this pr to verify this

It's okay. I found this when I added MultiClusterService API.

File: hack/verify-crdgen.sh (hunk 1 above)
Comment (Poor12) [reply hack/verify-crdgen.sh]: Done.

**Comment by RainbowMango** [comment]
> Sorry, I find the crd yaml of chart is out of date. Thinks for this pr to verify this

Fortunately, the missed content was just some comments, not affecting the functionality.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**